presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenState.kt

```diff
@@ -23,5 +23,7 @@ data class EventScreenState(
     val isStartDateTime: Boolean = false,
     val shouldOpenDropdown: Boolean = false,
     val alarmReminderItem: AlarmReminderItem = AlarmReminderItem.TEN_MINUTES,
-    val listOfAttendee: List<Attendee> = listOf()
+    val listOfAttendee: List<Attendee> = listOf(),
+    /** TODO I want to have an state that will update once the insertion has completed */
+    val hasInsertedCompleted: Boolean = false
```

Comment:
Is this correct

Reply:
This part Yes (Explained later)

Tho personally I like the name `isSaved` a bit more

Reply:
@MKiperszmid Updated